modules/swagger-codegen/src/main/resources/Java/apiInvoker.mustache

```diff
@@ -106,12 +106,12 @@ public class ApiInvoker {
 
     Builder builder = client.resource(host + path + querystring).accept("application/json");
     for(String key : headerParams.keySet()) {
-      builder.header(key, headerParams.get(key));
+      builder = builder.header(key, headerParams.get(key));
```

Comment:
I believe this is not needed, as `builder.header` sets the header value directly to the `builder` itself, right?

Please refer to https://jersey.java.net/nonav/apidocs/1.8/jersey/com/sun/jersey/api/client/PartialRequestBuilder.html#header(java.lang.String, java.lang.Object)